fleche/doc.ml

```diff
@@ -44,8 +44,14 @@ type t =
 
 let mk_doc root_state workspace =
   let libname = Names.(DirPath.make [ Id.of_string "foo" ]) in
-  let require_libs = [ ("Coq.Init.Prelude", None, Some (Lib.Import, None)) ] in
-  Coq.Init.doc_init ~root_state ~workspace ~libname ~require_libs
+  if !Coq.Workspace.no_init = true then
```

Comment:
@Alizter @artagnon this is bogus as `Workspace.apply` is called in `doc_init` below. That likely makes your trouble for HoTT.

Reply:
The right fix is to put require_libs in the workspace setup.